docs/node/daemon/config/settings.mdx

```diff
@@ -19,33 +19,47 @@ This section provides a detailed overview of the operational parameters availabl
+   - `pg_db_pass`: PostgreSQL database pass (may be omitted for some pg_hba.conf configurations).
 
-- `pg_db_name`: PostgreSQL database name (override database name). The default is `kwild`.
+   - `pg_db_name`: PostgreSQL database name (override database name). The default is `kwild`.
 
-- `private_key_path`: This variable specifies the path for node's private key. If left unspecified, it defaults to `./private_key` relative to the root directory.
+   - `private_key_path`: This variable specifies the path for node's private key. If left unspecified, it defaults to `./private_key` relative to the root directory.
 
-- `extension_endpoints`: This variable specifies the endpoints for extensions, separated by commas.
+   - `extension_endpoints`: This variable specifies the endpoints for extensions, separated by commas.
 
-- `tls_cert_file`: This variable specifies the file path containing a certificate to enable TLS on the gRPC server. This path can be either absolute or relative to the specified root directory. If unspecified, the gRPC server's communications remain unencrypted.
+   - `tls_cert_file`: This variable specifies the file path containing a certificate to enable TLS on the gRPC server. This path can be either absolute or relative to the specified root directory. If unspecified, the gRPC server's communications remain unencrypted.
 
-- `tls_key_file`: This variable specifies the file path containing the corresponding private key that enables TLS on the gRPC server. Like the `tls_cert_file`, this path can be either absolute or relative to the specified root directory. If unspecified, the gRPC server's communications remain unencrypted.
+   - `tls_key_file`: This variable specifies the file path containing the corresponding private key that enables TLS on the gRPC server. Like the `tls_cert_file`, this path can be either absolute or relative to the specified root directory. If unspecified, the gRPC server's communications remain unencrypted.
 
-- `hostname`: This variable sets the Kwil gRPC server's hostname.
+   - `hostname`: This variable sets the Kwil gRPC server's hostname.
+
+   - `snapshot_file`: This variable specifies the path to the snapshot file to be used for the initial state of the node. If the node is configured to start with initial state and this variable is not set, the node will fail to start.
+
+   ### [app.snapshots]
+
+      This section contains configurations for managing the snapshots of the kwild database.
+
+      - `enabled`: This variable toggles the snapshot feature on or off. By default, the snapshot feature is disabled.
+
+      - `snapshot_dir`: This variable specifies the path to the snapshots directory.
+
+      - `snapshot_heights`: This variable specifies the heights(multiples of heights) at which the database is snapshotted. The default is 10000, which is around 1 snapshot per day considering block rate of 1 block per 6-8 secs. 
```

Comment:
Missing space before the parenthesis.  I have noticed that you and Gavin do this often.  Is this a normal non-US style?  It's not the norm in US English, so it really stands out.

Reply:
I don't think so. I just never learnt the correct approach (more like not had taught) and it's hard now to look for patterns like these i guess,🤷‍♀️. Brennan had to keep correcting me very basic things and it's just hard to pick it up now